docs/source/asr/speaker_diarization/configs.rst

```diff
@@ -50,11 +50,17 @@ Diarizer Architecture Configurations
       model_path: null #.nemo local model path or pretrained model name or none
       window_length_in_sec: 0.15
       shift_length_in_sec: 0.01
-      threshold: 0.5 # tune threshold on dev set. Check <NeMo_git_root>/scripts/voice_activity_detection/vad_tune_threshold.py
       vad_decision_smoothing: True
       smoothing_params:
         method: "median" 
         overlap: 0.875
+      threshold: null # if not null, use this value and ignore the following onset and offset. Warning! it will be removed in release 1.5!
+      postprocessing_params: # Tune parameter on dev set. Check <NeMo_git_root>/scripts/voice_activity_detection/vad_tune_threshold.py
+        onset: 0.8
+        offset: 0.8
+        min_duration_on: 0.2 
+        min_duration_off: 0.2    
+        filter_speech_first: False 
```

Comment:
should this be an option? This helps in balancing FA and MISS, is this boolean value changing with datasets? for eg AMI and CH109?

Reply:
I made it to be an option particularly for convenience for experiments.  yes it could be diff with dataset. (I haven't had a concrete conclusion yet)